cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShowProceduresTest.scala

```diff
@@ -28,60 +28,158 @@ class ShowProceduresTest extends DocumentingTest {
+
+|m| mode
+|a| The procedure mode, for example `READ` or `WRITE`. label:default-output[]
+
+|m| worksOnSystem
+|a| Whether the procedure can be run on the `system` database or not. label:default-output[]
+
+|m| signature
+|a| The signature of the procedure.
+
+|m| argumentDescription
+|a| List of the arguments for the procedure, as map of strings with name, type, default, and description.
+
+|m| returnDescription
+|a| List of the returned values for the procedure, as map of strings with name, type, and description.
+
+|m| admin
+|a| `true` if this procedure is an admin procedure.
+
+|m| rolesExecution
+|a|
+List of roles permitted to execute this procedure.
+Will be `null` without the <<administration-security-administration-dbms-privileges-role-management,`SHOW ROLE`>> privilege.
+
+|m| rolesBoostedExecution
+|a|
+List of roles permitted to use boosted mode when executing this procedure.
+Will be `null` without the <<administration-security-administration-dbms-privileges-role-management,`SHOW ROLE`>> privilege.
+
+|m| option
+|a| Map of extra output, for example if the procedure is deprecated.
+||===
+""")
+    section("Syntax") {
+      p("""
+List all procedures::
+
+[source, cypher, role=noplay]
+----
+SHOW PROCEDURE[S]
+[YIELD { * | field[, ...] } [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+[WHERE expression]
+[RETURN field[, ...] [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+----
+[NOTE]
+====
+When using the `RETURN` clause, the `YIELD` clause is mandatory and may not be omitted.
+====
+
+List procedures that the current user can execute::
+
+[source, cypher, role=noplay]
+----
+SHOW PROCEDURE[S] EXECUTABLE [BY CURRENT USER]
+[YIELD { * | field[, ...] } [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+[WHERE expression]
+[RETURN field[, ...] [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+----
+
+[NOTE]
+====
+When using the `RETURN` clause, the `YIELD` clause is mandatory and may not be omitted.
+====
+
+List procedures that the specified user can execute::
+
+[source, cypher, role=noplay]
+----
+SHOW PROCEDURE[S] EXECUTABLE BY username
+[YIELD { * | field[, ...] } [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+[WHERE expression]
+[RETURN field[, ...] [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+----
+
+Required privilege <<administration-security-administration-dbms-privileges-user-management,`SHOW USER`>>.
+This command cannot be used for LDAP users.
+
+[NOTE]
+====
+When using the `RETURN` clause, the `YIELD` clause is mandatory and may not be omitted.
+====
+""".stripMargin('#'))
+    }
+    section("Listing all procedures") {
+      p(
+        """To list all available procedures with the default output columns, the `SHOW PROCEDURES` command can be used.
+          #If all columns are required, use `SHOW PROCEDURES YIELD *`.""".stripMargin('#'))
+      query("SHOW PROCEDURES", ResultAssertions(p => {
+        p.columns should contain theSameElementsAs Array("name", "description", "mode", "worksOnSystem")
```

Comment:
All descriptions render as code blocks and within quotation marks. It's not strictly wrong, but it doesn't look so nice.

Reply:
Its because that is the actual output